changelogs/1.23.0.yaml

```diff
@@ -33,6 +33,10 @@ minor_behavior_changes:
   change: |
     export symbols of LuaJit by default on Linux. This is useful in cases where you have a lua script
     that loads shared object libraries, such as those installed via luarocks.
+- area: admin
+  change: |
+    added &safe option to /stats?filter= to use RE2 regexes when filtering, which are much faster
+    than the default Ecmascript regex evaluator.
```

Comment:
I think the important detail isn't that it's Ecmascript, but that the std::regex implementation is terribly slow, so maybe say `than the default std::regex evaluator` or something.

Reply:
done